dangerflows/DangerCheck.ts

```diff
@@ -0,0 +1,292 @@
+/* eslint-disable @typescript-eslint/camelcase */
+// Provides dev-time type structures for  `danger` - doesn't affect runtime.
```

Comment:
I know, I know! It's just a space.. 😁 
```suggestion
// Provides dev-time type structures for `danger` - doesn't affect runtime.
```